.github/workflows/e2e-workflow-server.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         name: Checkout to Code
 
       - name: Run docker-compose stack
+        env:
```

Comment:
If I don't pass the `env` here, The tests are not able to get the `INDEX_PREFIX` and tests fails because of this. The `INDEX_PREFIX` is present in `conftests.py` as well.


what I might be missing here?